oat-sa/extension-tao-booklet
# Updated booklet generation flow

The task is [TAO-10414](https://oat-sa.atlassian.net/browse/TAO-10414)

Disabling the Preview button was done by following the next example https://github.com/oat-sa/extension-tao-test/pull/339/files

### How to test
An easier way to test it set up a normal queue, using RDS broker. 
```
php index.php 'oat\taoTaskQueue\scripts\tools\InitializeQueue' --broker=rds --persistence=default --receive=1 --queue=queue
```
After that need to run a command 
```
php index.php 'oat\taoTaskQueue\scripts\tools\RunWorker' --queue=queue --limit=1
```
that will give you the possibility run one task by one. Som you can catch state between tasks when label should be 'in progress'.
